### PR TITLE
text update for decals dr2 and sdss "lost set" data

### PR DESCRIPTION
**Pull request by CKrawczyk**
This should not be merged until the new data goes live (should be later today/tomorrow).

@willettk here are the text changes we talked about.  I was not sure if we wanted to change anything in the "History of..." on the front page, so I left it alone for now.  Let me know if anything should be changed.

### CODE REVIEW

File: app/lib/en.coffee [hunk 1]
```diff
@@ -1132,8 +1132,9 @@ module.exports =
...
+    p10: '''The images on the site today include our second set of DECaLS images, this time taken from the survey's second data release. This new data release covers a larger protion of the sky than the first. Finally, we have included a set of &quot;lost images&quot; from SDSS that should have been included in the Galaxy Zoo 2 sample. These images will fill in the hole that currently exists in the bright galaxy datat set.'''
```
Comment (willettk) [inline app/lib/en.coffee]: @CKrawczyk - small typo "datat" in last sentence.

**Comment by willettk** [comment]
I believe you also have to add the paragraphs to this file if you want them to appear: https://github.com/zooniverse/Galaxy-Zoo/blob/master/app/views/story.eco

Suggest deploying a local version of GZ (if you haven't already, instructions are in the README.md) to make sure any text changes are actually appearing. It's always mildly more complicated than I remember from just simple HTML.

**Comment by CKrawczyk** [comment]
OK, made those changes, I will give it a build locally tomorrow to make sure it works.

**Comment by CKrawczyk** [comment]
The changes show up in my local build without issue (after downgrading node).

File: app/lib/en.coffee [hunk 2]
```diff
@@ -1132,8 +1132,9 @@ module.exports =
...
+    p10: '''The images on the site today include our second set of DECaLS images, this time taken from the survey's second data release. This new data release covers a larger protion of the sky than the first. Finally, we have included a set of &quot;lost images&quot; from SDSS that should have been included in the Galaxy Zoo 2 sample. These images will fill in the hole that currently exists in the bright galaxy data set.'''
```
Comment (willettk) [inline app/lib/en.coffee]: "portion" is misspelled. 

Also - might be useful to add a date? Something like "starting in March 2016".

**Comment by willettk** [comment]
This should be reviewed and merged as soon as possible, since the new images are live.

**Pull request merged**